### PR TITLE
fix(python): update start page

### PR DESCRIPTION
**Pull request by Frzk**
Fix https://github.com/Scalingo/python-buildpack/issues/119

### CODE REVIEW

File: src/_posts/languages/python/2000-01-01-start.md [hunk 1]
```diff
@@ -22,17 +22,18 @@ The following versions of Python are available:
...
 | `3.9`          | up to `3.9.20`  | up to `3.9.20`  |
 | `3.8`          | up to `3.8.20`  | unsupported     |
 
 {% note %}
 Even though we still support them, we strongly advise against using deprecated
-versions of Python, such as Python `3.7.x`. If you are still using one of these
-deprecated versions, please consider migrating to a more recent version.
+versions of Python, such as Python `3.9.x` and below. If you are still using
+one of these deprecated versions, please consider migrating to a more recent
+version.
```
Comment (EtienneM) [inline src/_posts/languages/python/2000-01-01-start.md:36]: suggestion: I understand why the note here, but with the current information in the doc I don't understand how a customer would know what is a "deprecated" version. What do you think about removing the deprecated Python versions from the table above?

**Review by EtienneM — APPROVED** [review]
I disabled auto merge. I added a suggestion.

Already approving the PR. Feel free to merge as is if you want

File: src/_posts/languages/python/2000-01-01-start.md (hunk 1 above)
Comment (Frzk) [reply src/_posts/languages/python/2000-01-01-start.md]: I hesitated ^^
I've removed them, and I've also removed the warning in https://github.com/Scalingo/documentation/pull/2845/commits/eef5bba7df4bb36779e490cf402e0e509d3e3d84

Comment (EtienneM) [reply src/_posts/languages/python/2000-01-01-start.md]: Nice, I like when we are on the same page :D

**Review by EtienneM — APPROVED** [review]

**Pull request merged**